BParkingNano/python/muonsBPark_cff.py

```diff
@@ -11,12 +11,12 @@
                                  vertexCollection = cms.InputTag("offlineSlimmedPrimaryVertices"),
                                  
                                  ##for the output trigger matched collection
-                                 maxdR_matching = cms.double(0.01), #too tight 0.05 already better + check
+                                 maxdR_matching = cms.double(0.1),
                                  
                                  ## for the output selected collection (tag + all compatible in dZ)
                                  dzForCleaning_wrtTrgMuon = cms.double(1.),
 
-                                 ptMin = cms.double(1.),
+                                 ptMin = cms.double(0.5),
```

Comment:
do we really plan to go that low in muon pT ? Given that our signal is centrall and we need at least 3GeV to reach barrel do we gain anything?